src/library_html5.js

```diff
@@ -2672,6 +2672,23 @@ var LibraryJSEvents = {
     return requestAnimationFrame(tick);
   },
 
+  // TODO(sbc): USe MIN_XX_VERSION settings to avoid this polyfill on modern browsers.
+  $globalThis: function() {
+    // nop, used to pulli in globalThis__postset
+  },
+
+  $globalThis__postset: `
+    (function() {
+      if (typeof globalThis === 'object') return;
+      Object.prototype.__defineGetter__('__magic__', function() {
+        return this;
+      });
+      __magic__.globalThis = __magic__; // lolwat
```

Comment:
@mathiasbynens Look!

Reply:
I this a sensible thing to push in production.. i have no idea how this is working :)

Reply:
https://mathiasbynens.be/notes/globalthis